build/codegen/codegen.sh

```diff
@@ -52,3 +52,6 @@ find "${scriptdir}/../../pkg/client" -name "*.go" -exec \
 find "${scriptdir}/../../pkg/client" -name "*.go" -exec \
     $SED 's/cephNFSs/cephNFSes/g' {} +
 find "${scriptdir}/../../pkg/client" -name "*.go.bak" -delete
+
+# Generate and test the Rook Python client libraries
+"${scriptdir}"/rook-client-python/generate.sh
```

Comment:
Instead of documenting the dependency, could this script just print a readable error if it doesn't find the dependencies? As long as the tool is self-documenting, it seems sufficient instead of trying to find where to put documentation somewhere else.

Reply:
I like the idea.